manifest_staging/charts/aad-pod-identity/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "aad-pod-identity.fullname" . }}
+  labels:
+    {{- include "aad-pod-identity.labels" . | nindent 4 }}
+spec:
+  seLinux:
+    rule: RunAsAny
+  privileged: true
```

Comment:
MIC and NMI don't require the same set of privileges. We should give the least required privilege for both pods. So I think having 2 different policies bound to MIC and NMI service account is a good option. WDYT?

Reply:
Make totally sense, do you have suggestions? 
If you know exactly what permits they need, I'll fix the PR tomorrow morning.

Reply:
MIC:
- volumes:
  - hostPath
  - secret

NMI: 
- hostNetwork: true
- volumes:
  - hostPath
- allowedCapabilities
  - NET_ADMIN
  - DAC_READ_SEARCH
  - NET_RAW

Both: 
  - RunAsUser: 0 / privileged: true

Not too familiar with fsGroup and supplementalGroups. @aramase let me know if I miss any spec

Reply:
Done

Reply:
These are the narrowed down security context spec required:
For MIC: https://github.com/Azure/aad-pod-identity/pull/998#discussion_r595487711
For NMI: https://github.com/Azure/aad-pod-identity/pull/998#discussion_r595489291

Reply:
done